engines/ultima/ultima8/debugtools.cpp

```diff
@@ -0,0 +1,348 @@
+						// ImGui::TableSetupColumn("Value", ImGuiTableColumnFlags_WidthFixed);
+						// ImGui::TableHeadersRow();
+
+						ImGui::TableNextColumn();
+						ImGui::Text("Id");
+						ImGui::TableNextColumn();
+						ImGui::Text("%u", item->getObjId());
+						ImGui::TableNextColumn();
+						ImGui::Text("Class");
+						ImGui::TableNextColumn();
+						ImGui::Text("%s", item->GetClassType()._className);
+						ImGui::TableNextColumn();
+						ImGui::Text("Shape");
+						ImGui::TableNextColumn();
+						ImGui::Text("%u", item->getShape());
+						ImGui::TableNextColumn();
+						ImGui::Text("Frame");
+						ImGui::TableNextColumn();
+						ImGui::Text("%u", item->getFrame());
+						ImGui::TableNextColumn();
+						ImGui::Text("Map");
+						ImGui::TableNextColumn();
+						ImGui::Text("%u", item->getMapNum());
+						ImGui::TableNextColumn();
+						ImGui::Text("Location");
+						ImGui::TableNextColumn();
+						if (item->getParent()) {
+							int32 gx, gy;
+							item->getGumpLocation(gx, gy);
+							ImGui::Text("(%d, %d)", gx, gy);
+						} else {
+							Point3 p = item->getLocation();
+							ImGui::Text("(%d, %d, %d)", p.x, p.y, p.z);
+						}
+						ImGui::TableNextColumn();
+
+						ImGui::Text("Footpad");
+						ImGui::TableNextColumn();
+						int32 xd, yd, zd;
+						item->getFootpadData(xd, yd, zd);
+						ImGui::Text("%d, %d, %d", xd, yd, zd);
+						ImGui::TableNextColumn();
+						// Original weights appear to be different
+						ImGui::Text("Weight");
+						ImGui::TableNextColumn();
+						ImGui::Text("%d", item->getWeight());
+						ImGui::TableNextColumn();
+						ImGui::Text("Volume");
+						ImGui::TableNextColumn();
+						ImGui::Text("%d", item->getVolume());
+						ImGui::TableNextColumn();
+						// Original menu had "Anim" here - None, Unk, Normal, Fast
+						// Original menu had "Fr" and "Sp" here
+						ImGui::Text("Family");
+						ImGui::TableNextColumn();
+						ImGui::Text("%d", item->getFamily());
+						ImGui::TableNextColumn();
+						ImGui::Text("Usecode");
+						ImGui::TableNextColumn();
+						const char *ucname = GameData::get_instance()->getMainUsecode()->get_class_name(item->getShape());
+						ImGui::Text("%s", ucname != nullptr ? ucname : "");
+						ImGui::TableNextColumn();
+						ImGui::Text("Quality");
+						ImGui::TableNextColumn();
+						ImGui::Text("%u", item->getQuality());
+						ImGui::TableNextColumn();
+						ImGui::Text("NPC Number");
+						ImGui::TableNextColumn();
+						ImGui::Text("%u", item->getNpcNum());
+						ImGui::TableNextColumn();
+						ImGui::Text("Equipment Type");
+						ImGui::TableNextColumn();
+						ImGui::Text("0x%x", si ? si->_equipType : 0);
+
+						ImGui::EndTable();
+					}
+				}
+
+				// ShapeInfo "Type Flags"
+				Common::String shapeFlagsLabel = Common::String::format("Shape Flags: 0x%04x", si->_flags);
+				if (ImGui::CollapsingHeader(shapeFlagsLabel.c_str(), ImGuiTreeNodeFlags_DefaultOpen)) {
```

Comment:
I think this might be changing the id for the header and need a ###id format attached to the label